[ADD] account_bank_statement_clear_partner (#322)
OCA/bank-statement-import: account_bank_statement_clear_partner/readme/USAGE.rst

To use this module, you need to:

> #. Go to **Invoices** > **Bank**

Reviewer: These instructions are not correct, as that's not the correct menu path (Invoicing is the main menu, but then explained about the dashboard. You need also to explain - probably in configuration section -, that for having such dashboard, you need to enable "Show Full Accounting Features" in your user).

#. Select a bank statement

> #. Press **Clear partners** and all the partners in reconciled lines will be cleared.

Reviewer: ```suggestion
#. Press "Clear partners" button located on the header.
#. All the partners in non reconciled lines will be cleared.
```

To use this module, you need to:

#. Enable the **Show Full Accounting Features** option for your user under **Settings** > **Users & Companies** > **Users**

> #. Go to **Invoices**

Reviewer: ```suggestion
#. Go to **Invoicing**
```

> #. Select **Bank**

Reviewer: ```suggestion
#. Click on the name of a bank journal (**Bank** in demo data for example).
```
